client/components/jetpack/daily-backup-status/status-card/backup-in-progress.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { ProgressBar } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { Moment } from 'moment';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'calypso/lib/formatting';
+import { INDEX_FORMAT } from 'calypso/lib/jetpack/backup-utils';
+import { backupMainPath } from 'calypso/my-sites/backup/paths';
+import getSelectedSiteSlug from 'calypso/state/ui/selectors/get-selected-site-slug';
+import useGetDisplayDate from '../use-get-display-date';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import cloudScheduleIcon from './icons/cloud-schedule.svg';
+
+const BackupInProgress: React.FC< Props > = ( { percent, inProgressDate, lastBackupDate } ) => {
+	const translate = useTranslate();
+	const getDisplayDate = useGetDisplayDate();
+
+	const siteSlug = useSelector( getSelectedSiteSlug );
+
+	// The backup 'period' is a Unix timestamp in terms of seconds
+	const inProgressDisplayDate = getDisplayDate( inProgressDate, false );
+	const lastBackupDisplayDate = lastBackupDate
```

Comment:
In `client/components/jetpack/backup-card/backup-in-progress.tsx`, we apply to `lastBackupDate` the `useDateWithOffset` hook before using the variable. Here we don't do that. Could we be inadvertently showing two different dates depending on the version of the component?